layout/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "frameworks": {
+    "net461": {
+      "dependencies": {
+        "Microsoft.TargetingPack.NETFramework.v4.6.1": "1.0.1"
+      }
+    },
+    "netcoreapp1.1": {
+      "dependencies": {
+        "NETStandard.Library": "1.6.2-beta-24709-02"
```

Comment:
This one isn't used any anywhere here it was leftover from when I moved this from the standard repo. I will remove.